abc.templates/README.md

```diff
@@ -0,0 +1,18 @@
+# GitHub Token Minter
+
+## Installation with abc CLI
+
+1. Render infrastructure template and actuate terraform.
+```
+abc templates render \
+--dest infra/github-token-minter \
+--input=automation_service_account_member=<SERVICE_ACCOUNT> \
+--input=project_id=<PROJECT_ID> \
+--input=domains=<COMMA_SEPARATED_DOMAINS> \
+--input=terraform_state_bucket=<TERRAFORM_STATE_BUCKET> \
+--input=terraform_state_prefix=<TERRAFORM_STATE_PREFIX> \
+--input=github_owner_id=<GITHUB_ORG_ID> \
+github.com/abcxyz/github-token-minter.git//abc.templates/infra?ref=<SHA>
```

Comment:
I think as a best practice, this should be a version, not necessarily a SHA (it could be) but for readability and human understanding installing based on a version tag might be cleaner.

Reply:
I agree - we could do `TAG_OR_SHA`